src/style.css

```diff
@@ -2,2 +1,97 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+h1 {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+}
+
+header {
+  width: 100%;
+  background-color: #fff;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
+}
+
+.logo img {
+  height: 40px;
+}
+
+nav ul li a {
+  display: inline-block;
+  text-decoration: none;
+  color: #000;
+  text-align: center;
+  line-height: 60px;
+  height: 60px;
+  transition: color 0.3s ease;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+}
+
+nav ul {
+  display: flex;
+  list-style: none;
+}
+
+nav ul li {
+  margin-left: 20px;
+}
+
+nav ul li a:hover {
+  color: #00acdc;
```

Comment:
Consider using a CSS variable for the color #00acdc to improve maintainability and consistency across your styles.